roles/kubernetes-apps/persistent_volumes/openstack/templates/openstack-storage-class.yml.j2

```diff
@@ -7,9 +7,19 @@ metadata:
   annotations:
     storageclass.kubernetes.io/is-default-class: "{{ class.is_default | default(false) | ternary("true","false") }}"
 provisioner: kubernetes.io/cinder
+mountOptions:
```

Comment:
HI @Payback159 

Is it better to skip this line if the `mountOptions` is not define. like:
```
{% if class.mountOptions is defined %}
XXX
{% endif %}
```

Reply:
Hi @yankay ,

I have tested it in a private installation at our company. If we don't remove the mountOption then there would be a line mountOption in the yaml file (template) but the Kubernetes API accepts the file without problems.

From my point of view a trade off of "extra logic" in the template versus a clean yaml file for rolling out the StorageClass.

I think it would be more logical if I still implement your suggestion. How do you see this?

Reply:
HI @Payback159 

Yes, my suggestion before is to make the code more readable and logical. like : https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes/node/templates/kubelet-config.v1beta1.yaml.j2#L132

Reply:
I agree with you, I have adjusted the logic.